.github/workflows/run_tests.yml

```
@@ -67,15 +57,11 @@ jobs:
 
-      - name: Setup Python
-        uses: actions/setup-python@v6
-        with:
-          python-version: "3.12"
-
       - name: Install uv
         uses: astral-sh/setup-uv@v7
         with:
-          version: "0.6.5"
+          version: "0.12.0"
+          python-version: ${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] property "python-version" is not defined in object type {}

(expression)